### DISCUSSION TITLE
リリース申請

### DISCUSSION THREAD

**Discussion by ghost** [lit-iphone-course/lit-iphone-course#65] [Question-original] [answered]
(1) 実現したいこと
リリース申請・ビルドの提出
(2) 実現するために自分が試した内容とその結果
教科書・バンドルIDが間違っていないか確認
(3) 調べた内容や資料

(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)

(5) GitHubのリンク
https://github.com/j-fuuuu/frognote_original.git

**Comment by txxxxc** [comment]
Slackに貼ってたスクショ添付してもらえるとありがたいです！！

**Comment by txxxxc** [comment, marked as answer]
なんかApple側のバグらしいので同じ手順でもう一回提出してみて良さそうですー！
> Hi All, I've tried again the same archive to be uploaded without any change and it worked, so it's true it was something in the back-end validation from apple. 

参考：https://developer.apple.com/forums/thread/707350

**Reply by ghost** [reply]
もう一度提出して同じエラーが出ましたが、appleからメールでビルド処理完了のメールがきたので、多分できてると思われます。
<img width="1440" alt="スクリーンショット 2022-09-27 11 55 46" src="https://user-images.githubusercontent.com/111028272/192668337-eb71100b-0aca-41d0-8de8-29d5de1bc496.png">

**Reply by txxxxc** [reply]
あ、スクショ貼れてないっぽいのでもう一回貼ってほしいですー！

**Reply by ghost** [reply]
貼り直しました。

ビルドの提出、できました。すみません、ありがとうございました。

**Reply by txxxxc** [reply]
あ、よかったです～！

**Reply by txxxxc** [reply]
ちなみにどうやって解決した？？

**Reply by ghost** [reply]
<img width="934" alt="スクリーンショット 2022-09-28 13 13 58" src="https://user-images.githubusercontent.com/111028272/192686189-aaa80f3e-73d0-4f97-8045-271bde90f3f0.png">

このエラーが出たあとに、appleからこのメールが送られてきて、できるようになっていました。

**Reply by txxxxc** [reply]
なるほど！共有ありがとうー！🙏🙏🙏
